pages/transactions.js

```diff
@@ -106,4 +213,15 @@ class TransactionsPage extends React.Component {
   }
 }
 
-export default withUser(addCollectiveCoverData(TransactionsPage));
+const addTransactionsData = graphql(transactionsQuery, {
+  skip: props => !props.slug,
```

Comment:
`collectiveSlug` is required in https://github.com/opencollective/opencollective-frontend/blob/a34c29ae3056d356bf760664220b4efedbce6821/server/pages.js#L41. Is this check still necessary?